kombu/transport/redis_cluster.py

```diff
@@ -27,15 +28,18 @@
 logger = get_logger(__name__)
 
 
-# Override this method to use other redis client
-def create_redis_cluster_connection(hostname, port, password, ssl):
+# Override create_redis_cluster_connection_for_{producer,consumer} to use other redis client
+def create_redis_cluster_connection_for_consumer(hostname, port, password, ssl):
     params = {'skip_full_coverage_check': True, 'host': hostname, 'port': port, 'password': password}
```

Comment:
`skip_full_coverage_check` was used in `redis-py-cluster`. use `require_full_coverage` instead with a reversed value

Reply:
`dynamic_startup_nodes` should be there, since we are using a DNS endpoint as a startup node

Reply:
Is it possible for Soda to pass parameters into this function? We should be able to add/remove/tune parameters, but the current structure requires releasing kombu every time

Reply:
In soda, we override entire method to use soda's RedisCluster(https://github.com/sendbird/soda/blob/develop/soda/celery.py#L67), so it would be fine to keep parameter as-is.

I'll change default parameters, Thanks!